introduction.md

```diff
@@ -1,3 +1,4 @@
 Introduction
 ============
-My favourite path is, with no doubt, the JavaScript path!
+*My favourite path is, with no doubt, the JavaScript path!
+*Git path is also gorgerous!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?